frontend/src/services/AuthService.ts

```diff
@@ -32,5 +34,9 @@ export const verifyLoginOtp = async (params: {
 }
 
 export const logout = async (): Promise<void> => {
+  // Remove logged in state from localStorage
+  localStorage.removeItem(LOGGED_IN_KEY)
+  // Event to let useLocalStorage know that key is being deleted.
+  window.dispatchEvent(new Event(LOCAL_STORAGE_EVENT))
```

Comment:
/justANote:
Not from your PR, but the working of `LOCAL_STORAGE_EVENT` and `useLocalStorage` seems a bit funny and inefficient 🤔. Every component which use `useLocalStorage` (and specifies the key it cares about) take an action on all key changes. [Storage events](https://developer.mozilla.org/en-US/docs/Web/API/Window/storage_event) by right contain information about what changed, so listeners can take informed decisions. 

Not a blocker to merge, maybe I'll have a chat with KarRui later.